### PR TITLE
Update requirements

### PR DESCRIPTION
**Pull request by ethanhs**
*This pull request was made automatically by a human*

### CODE REVIEW

**Comment by ethanhs** [comment]
This is actually stalled on updating ocflib, so I'll do that now.

File: ircbot/db.py [hunk 1]
```diff
@@ -5,16 +5,17 @@
...
+        host='mysql.ocf.berkeley.edu',
+        cursorclass=pymysql.cursors.DictCursor,
+        charset='utf8mb4',
+        autocommit=True,
+    )
+    try:
+        with conn.cursor() as cursor:
             yield cursor
+    finally:
+        conn.close()
```
Comment (cg505) [inline ircbot/db.py:21]: what is the point of using a `with` expression if you still have to do this?

File: ircbot/ircbot.py [hunk 2]
```diff
@@ -146,7 +147,10 @@ def __init__(
 
     def register_plugins(self):
         for importer, mod_name, _ in pkgutil.iter_modules(['ircbot/plugin']):
-            mod = importer.find_module(mod_name).load_module(mod_name)
+            assert isinstance(importer, PathEntryFinder)
+            loader = importer.find_module(mod_name)
+            assert loader is not None
```
Comment (cg505) [inline ircbot/ircbot.py:152]: why are these `assert`s now necessary?

**Review by cg505 — APPROVED** [review]
just a couple questions

File: ircbot/db.py (hunk 1 above)
Comment (ethanhs) [reply ircbot/db.py:21]: The with is used to manage the context of the cursor, the connect/close doesn't have a contextmanager :/

File: ircbot/ircbot.py (hunk 2 above)
Comment (ethanhs) [reply ircbot/ircbot.py:152]: These are assertions to please the type checker which can't tell that `importer` is always a `PathEntryFinder` and unless importing is broken the loader shouldn't be `None`.

**Pull request merged**